iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,30 +1,34 @@
-• New OSM maps data as of September 12
-• Fixed crash on startup after editing some OSM objects
-• Bookmark lists are now sorted by recently modified time
-• Fixed annoying "Continue detecting location" dialog when returning to OM
-• Improved displaying of the next street name directions
-• Use the same system font everywhere
-• Use the default mail app on iOS 14 to report a bug
-• Enabled Catalan translations
+• New OSM data as of October 19
+• Minor performance improvements
+• Dump corrupted KML to the log on startup
+• Do not strip non-ASCII characters from exported bookmarks
+• Simplify/smooth road altitudes chart
+• Use system 12/24h time format in the Place Page
+
+Routing and Navigation:
+• Show "Roads to avoid" options for walking and cycling routes
+• Update vehicle models
+• Show warnings on the route for all router types
+• Enable Catalan TTS
+• Show speed camera on tap
 
 Search:
-• Show features in the viewport at the top of search results
-• Significantly speedup some search scenarios
-• Improved category ranking & address search.
-• Fixed highlighted hotel search results icon on the map
-• Improved German category search
+• Show all found results on the map screen
+• Improved villages search
+• Show only matched categories if they were selected from the list, not names
+• Fixed street synonyms for nordic languages
 
-Styles and icons:
-• Added driving_school, music_school, language_school, archaeological_site, craft-painter
-• Removed backgrounds from historic cemetery icons
-• Added different castle types: castrum, fortified_church, fortress, hillfort, kremlin, manor, palace & shiro
-• Increased visibility of education & some sports
-• Separated furniture & kitchen types
-• Better looking leisure=track
-• Fixed vending=fuel icon
+Styles:
+• Use regional metro/subway icons for entrances and stations
+• Add brewery, plumber, tailor, carpenter, sawmill, beekeeper, blacksmith, electrician, painter and other craft type icons
+• Fixed historic=* priority
+• Draw international airport icon on z7-9
+• Draw metro for major cities starting from z12
+• Added natural=shrubbery, natural=desert, leisure=dance, amenity=charging_station-motorcar, and more barrier types
+• Do not draw railway station icon for building=train_station
+• Updated icons for garden, park, forest, national park, nature reserve, power, chimney
+• Add historic gallows, pillory, memorial cross, stolperstein and war_memorial
 
-Localizations:
-• Added missing translations for craft=* and residential gardens
-• Added/fixed Dutch, Portuguese (Brazil), Italian, Catalan, Turkish, Belarusian translations
-• Enabled Catalan TTS (Text-To-Speech)
-• Localized some Telegram and Instagram links
+Translations:
+• Use larger, more geographically significant names for Oregon maps in Downloader
+• Updated Brasil, Catalan, Chinese, Dutch, Finnish, German, Italian, Polish, Portuguese, Turkish translations
```

Comment:
Updated Catalan, Chinese, Dutch, Finnish, German, Italian, Polish, Portuguese (European and Brazilian), Turkish translations

Reply:
(or just Portuguese)